modules/wilson_content/wilson_content.module

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @file
+ * Override the Wilson content editing experience.
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Implements hook_entity_view_alter().
+ */
+function wilson_content_entity_view_alter(&$build, EntityInterface $entity, $display) {
```

Comment:
It would be nice to get in the habit of typehinting arguments more. In this instance, see: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21entity.api.php/function/hook_entity_view_alter/9.4.x 

```
function [hook_entity_view_alter](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21entity.api.php/function/hook_entity_view_alter/9.4.x)(array &$build, \Drupal\Core\Entity\EntityInterface $entity, \Drupal\Core\Entity\Display\EntityViewDisplayInterface $display)
```

Reply:
Just a query on this, can I use `use` statements to avoid having the full namespaces for the classes in the function arguments?

i.e.

```
use Drupal\Core\Entity\EntityInterface;
use Drupal\Core\Entity\Display\EntityViewDisplayInterface;

/**
 * Implements hook_entity_view_alter().
 */
function wilson_entity_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
  // Override the map zoom display settings with the value of field_map_zoom.
  if ($entity->hasField('field_map_address') && $entity->hasField('field_map_zoom')) {
    $zoom = $entity->get('field_map_zoom')->getString();
    $build['field_map_address'][0]['#zoom'] = $zoom;
  }
}
```

Reply:
Yep absolutely :D (and definitely the preferred way to do it!)